project.php

```diff
@@ -1763,17 +1763,29 @@ function echo_download_zip( $link_text, $discriminator )
 
         $url = "$project->url/$p";
         $filesize_b = filesize("$project->dir/$p");
+        $last_modified = filemtime("$project->dir/$p");
     }
 
     echo "<li>";
     echo "<a href='$url'>";
     echo $link_text;
     echo "</a>";
     echo_byte_size($filesize_b);
+    echo_last_modified($last_modified);
     echo "</li>";
     echo "\n";
 }
 
+function echo_last_modified($last_modified)
+{
+    global $datetime_format;
+
+    if (isset($last_modified))
+    {
+        echo " (", strftime($datetime_format, filemtime($file)), ")";
```

Comment:
`s/filemtime($file)/$last_modified/`

Reply:
This is what happens when I do a small refactor after testing but before pushing like a fool.

Reply:
I assumed as much - it happens, don't worry about it! 😁

Reply:
Join the club. 😁